src/components/Modals/FilterModal/FilterModal.scss

```diff
@@ -8,6 +8,7 @@
         top: -4rem;
         z-index: 1;
         height: calc(100vh - 8rem);
+        width: 100%;
```

Comment:
Filter modal width on mobile looked odd, cause the width was set
![Screenshot 2024-10-07 at 3 48 01 PM](https://github.com/user-attachments/assets/2aa8d77d-4027-4753-9668-cf4b77de175f)